docs/vm-ee-guide.md

```diff
@@ -0,0 +1,267 @@
+
+```text
+# Output
+Name:         i-0c2e4054f435927a9
+Namespace:    sample-ns
+Labels:       nephe.io/cloud-vm-uid=i-0c2e4054f435927a9
+              nephe.io/cloud-vpc-uid=vpc-0882e6544e1bb0564
+              nephe.io/cpa-name=cloudprovideraccount-aws-sample
+              nephe.io/cpa-namespace=sample-ns
+              nephe.io/vpc-name=vpc-0882e6544e1bb0564
+Annotations:  <none>
+API Version:  runtime.cloud.antrea.io/v1alpha1
+Kind:         VirtualMachine
+Metadata:
+  Creation Timestamp:  <nil>
+  UID:                 938d3805-faa9-48f3-9e54-20a19c1f3d55
+Status:
+  Agented:         false
+  Cloud Id:        i-0c2e4054f435927a9
+  Cloud Name:      ubuntu-vm
+  Cloud Vpc Id:    vpc-0882e6544e1bb0564
+  Cloud Vpc Name:  test-vpc
+  Network Interfaces:
+    Ips:
+      Address:       10.10.6.108
+      Address Type:  InternalIP
+      Address:       54.219.53.133
+      Address Type:  ExternalIP
+    Mac:             06:3b:b6:bf:e2:9d
+    Name:            eni-06dc156b39555a674
+  Provider:          AWS
+  Region:            us-west-1
+  State:             running
+  Tags:
+    Name:  ubuntu-vm
+Events:    <none>
+```
+
+#### Azure
+
+```bash
+kubectl describe vm vm1-11610 -n sample-ns
+```
+
+```text
+# Output
+Name:         vm1-11610
+Namespace:    sample-ns
+Labels:       nephe.io/cloud-vm-uid=d33fa6ce-9a12-48bd-ac0e-d842a2f24838
+              nephe.io/cloud-vpc-uid=c440672a-47eb-462c-a567-4c6bc75eaabe
+              nephe.io/cpa-name=cloudprovideraccount-azure-sample
+              nephe.io/cpa-namespace=sample-ns
+              nephe.io/vpc-name=test-vnet-13196
+Annotations:  <none>
+API Version:  runtime.cloud.antrea.io/v1alpha1
+Kind:         VirtualMachine
+Metadata:
+  Creation Timestamp:  <nil>
+  UID:                 3a7f4cbc-ab54-4a34-890c-51419a7b64a2
+Status:
+  Agented:         false
+  Cloud Id:        /subscriptions/58bbb0ce-e26a-4bbc-a49c-284f63c25e2e/resourcegroups/test-rg/providers/microsoft.compute/virtualmachines/vm1
+  Cloud Name:      vm1
+  Cloud Vpc Id:    /subscriptions/58bbb0ce-e26a-4bbc-a49c-284f63c25e2e/resourcegroups/archana-rg/providers/microsoft.network/virtualnetworks/test-vnet
+  Cloud Vpc Name:  test-vnet
+  Network Interfaces:
+    Ips:
+      Address:       10.0.0.4
+      Address Type:  InternalIP
+      Address:       20.245.116.185
+      Address Type:  ExternalIP
+    Mac:             00-22-48-0A-00-8E
+    Name:            /subscriptions/58bbb0ce-e26a-4bbc-a49c-284f63c25e2e/resourcegroups/archana-rg/providers/microsoft.network/networkinterfaces/vm1156
+  Provider:          Azure
+  Region:            westus
+  State:             running
+  Tags:
+    Cloud:  azure
+    Name:   vm1
+Events:              <none>
+```
+
+## ExternalEntity
+
+For each imported `VirtualMachine`, an `ExternalEntity` CR is created. All the labels and tags from the `VirtualMachine`
+objects are imported on to `ExternalEntity` CR as labels, which can be used to configure ANPs.
+
+### Sample ExternalEntity Output
+
+#### Azure
```

Comment:
Missing AWS output here.

Reply:
I added again, let me know what you think about it.